.github/ISSUE_TEMPLATE/site_update.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Site Update
+about: Suggest an update to the GetNighthawk.dev website
+title: ''
+labels: 'component/ui'
```

Comment:
```suggestion
labels: 'component/UI'
``` @leecalcote looks better?

Reply:
@leecalcote looks better?

Reply:
@subhamkrai thanks for this. Yes, in general, you're right - it does. A challenge here is that the other 44 repos in the `layer5io` org also use the lowercased version. Changing it here may break label searches across the repos.

Reply:
Okay then it makes sense to use lowercase.

Reply:
Approved😀

Reply:
Awesome. Thank you for considering this, @subhamkrai. It is attention to this type of detail that soothes my OCD.

Reply:
Thanks